client/lib/directly/index.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * External dependencies
+ */
+import { noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+// import { loadScript } from 'lib/load-script';
+
+const DIRECTLY_SCRIPT_PATH = 'https://www.directly.com/widgets/rtm/embed.js';
+const DEFAULT_RTM_WIDGET_OPTIONS = {
+	// Automattic widget ID
+	id: '8a2968fc57d1e2f40157f42bf2d43160',
+	// By default don't show the "Ask a Question" button
+	displayAskQuestion: false
+};
+
+let initialized = false;
+
+const loadDirectlyScript = ( callback = noop ) => {
+	// It would be great to use lib/load-script here, however the Directly script requires
+	// the <script> tag to have id="directlyRTMScript", otherwise it fails hard. So this is
+	// a stripped-down version of lib/load-script with the id added.
+	function handleCompletedRequest( event ) {
```

Comment:
So the answer is... kinda? Here's the expanded relevant snippet of `embed.js` that relies on that id:

```
r.prototype.setBaseUrl = function(){
	var t = e.getElementById("directlyRTMScript"),
			i = e.createElement("a"),
			n = "";
	i.href = t.src,
	n += i.protocol,
	n += "//",
	n += i.hostname,
	n += i.port ? ":" + i.port : "",
	this.baseUrl = n
}
```

tl;dr It pulls the `src` attribute from the `#directlyRTMScript`, to use the same base URL for its other assets (CSS and iframe content). So the options I see as a workaround include:

1. Add an element to the page with a `src` attribute that has the same base URL as the embed script. It looks like most elements with valid `src` attributes will have side-effects (e.g. `<img id="directlyRTMScript" src="https://www.directly.com">` will try to download the image). Looks like we can use `<source>` without anything auto-downloading (though it smells bad to do this), or we could throw caution to the wind and do `<div id="directlyRTMScript" src="https://www.directly.com"></div>`). Both of these are invalid and I don't know if quirky browsers will bork, so more testing is required.

2. Self-host `embed.js` ([as suggested on this PR](https://github.com/Automattic/wp-calypso/pull/11306/files#r100466758)) and rewrite the above-pasted method, e.g. by hardcoding it to set the base URL of "https://www.directly.com".

Any other ideas?